### ISSUE TITLE
webpack2 module fails lint checks

### ISSUE THREAD

**Issue by mjs510** [marionettejs/marionette-integrations#46]
When running `npm run build` in the webpack2 module, eslint throws parsing errors:

```
/data/git/identity/ui-tests/webpack2/app/components/App.js
  1:1  error  Parsing error: The keyword 'import' is reserved

/data/git/identity/ui-tests/webpack2/app/components/ItemView.js
  1:1  error  Parsing error: The keyword 'import' is reserved

/data/git/identity/ui-tests/webpack2/app/initialize.js
  1:1  error  Parsing error: The keyword 'import' is reserved
```

This prevents the build from completing.

**Comment by scott-w** [comment]
Hi @mjs510 thanks for reporting this.

I looks like the linter config needs to be updated to account for ES6 in this module.

**Comment by punk73** [comment]
just in case someone facing the same problem and don't understand @scott-w answer just like me.

you need to open package.json and changing `"lint": "eslint app",`  to   `"lint": "eslint account",`

**Comment by jeffreycahyono** [comment]
@punk73 don't change`"lint": "eslint app"` . it tells eslint the folder should be linted.
Solution: 

1. babel-eslint should be installed using `npm instal --save-dev babel-eslint`
2. create .eslintrc which contents is:
```
{
    "parser": "babel-eslint"
}
```

**Comment by paulfalgout** [comment]
Oops thought the latest webpack2 PR addressed this issue, but the webpack only build without mocha wasn't addressed

**Comment by jeffreycahyono** [comment]
Sorry for skipping the webpack2 folder, here is the fix   #48
